[Device capabilities][1797963][People picker control] (#3449)
MicrosoftDocs/msteams-docs: msteams-platform/concepts/device-capabilities/people-picker-control-capability.md

---

> title: Integrate People Picker control capability

Reviewer: People Picker control capability is a bit confusing since "control" and "capability" both convey similar meanings. So, let's stick with "People Picker capability" in all places, including the page URL. This also aligns with our existing naming convention of using "capability" at end of the features.

author: Rajeshwari-v

> description: How to use Teams JavaScript client SDK to integrate People Picker control capability

Reviewer: ```suggestion
description: How to use Teams JavaScript client SDK to integrate People Picker capability
```

keywords:  people picker control
ms.topic: conceptual
localization_priority: Normal
ms.author: surbhigupta
---

# Integrate People Picker control capability 

> People Picker is a control to search and pick people. This is a native capability available in Teams platform. You can integrate Teams’ native People Picker input control with your web apps. You can define search context, single or multi selection, and other configuration to search people within a chat, groups, channels, teams, and across the organization with AAD user list.

Reviewer: ```suggestion
People Picker is a control to search and select people. This is a native capability available in Teams platform. You can integrate Teams’ native People Picker input control with your web apps. You can choose between single or multi-selection, and other configurations such as limiting search within a chat/channels, or across the entire organization.
```

> You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides `People Picker` API to integrate the People Picker control capability within your web app. 

Reviewer: ```suggestion
You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides `People Picker` API to integrate the People Picker control within your web app. 
```

## Advantages of integrating People Picker control capability

> * `People Picker` API enables you to add the People Picker control to your web apps, which  surfaces in a task module, as a tab or personal app. 

Reviewer: ```suggestion
* The People Picker control works in all of Teams surfaces - task module, a chat/channel/meeting tab, and personal app. 
```

* This cross platform control allows users to select entities, such as users in a group, chat, channel.
* The People Picker control capability helps with task assignment, tagging, notifying a user, and so on. 

You must call the `People Picker` API to integrate People Picker control capability. For effective integration, you must have an understanding of [code snippet](#code-snippet) for calling the `People Picker` API. 

> It is important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your web app.

Reviewer: ```suggestion
* This control allows users to search for and select users within a chat or channel, or even the entire organization
* The control saves significant effort on part of the developers as they no longer have to deal with the complexities of building such a control on their own

You must call the `People Picker` API to integrate People Picker control in your Teams app. For effective integration, you must have an understanding of [code snippet](#code-snippet) for calling the `People Picker` API. 
It is important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your web app.
```
